test/e2e/suite/command/blob/sign.go

```diff
@@ -26,27 +26,27 @@ import (
 
 const tsaURL = "http://timestamp.digicert.com"
 
-var _ = Describe("notation blob sign", func() {
+var _ = Describe("notation blob sign", Serial, func() {
```

Comment:
```suggestion
var _ = Describe("notation blob sign", func() {
```
If we create signatures under `vhost` directly, all test cases are isolated, and we can run them in parallel to improve performance.